slack-base/src/main/java/com/hubspot/slack/client/models/blocks/ActionsIF.java

```diff
@@ -0,0 +1,26 @@
+package com.hubspot.slack.client.models.blocks;
+
+import java.util.List;
+
+import org.immutables.value.Value;
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.models.blocks.elements.BlockElement;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface ActionsIF extends Block {
+  String TYPE = "actions";
+
+  @Override
+  @Value.Default
+  default String getType() {
+    return TYPE;
+  }
+
+  List<BlockElement> getElements();
```

Comment:
Definitely `List` and not `Set`?

Reply:
No - it's acceptable that you may have exactly the same block multiple times (there's no unique id, and you may want to show exactly the same block later in the message).

Reply:
Also I suspect you want these in order, right? Since set makes no guarantees that seems like it would be risky